Entities/Items/Crate/Crate.as

```diff
@@ -328,6 +332,9 @@ void onCommand(CBlob@ this, u8 cmd, CBitStream @params)
 			{
 				this.set_u32("unpack time", getGameTime() + this.get_u32("unpack secs") * getTicksASecond());
 				this.getShape().setDrag(10.0f);
+
+				CBlob @caller = getBlobByNetworkID( params.read_u16() );
+				if (caller !is null) this.server_setTeamNum( caller.getTeamNum() );
```

Comment:
Most of the time you should put if statements over multiple lines like so to avoid lines from getting too long. Exceptions include short if statements, the most common being `if (x is null) return;`.
```suggestion
				if (caller !is null)
				{
						this.server_setTeamNum(caller.getTeamNum());
				}
```